habitat-lab/habitat/tasks/rearrange/actions/grip_actions.py

```diff
@@ -37,7 +37,7 @@ def action_space(self):
 
     def _grasp(self):
         scene_obj_pos = self._sim.get_scene_pos()
-        ee_pos = self.cur_robot.ee_transform.translation
+        ee_pos = self.cur_robot.ee_transform().translation
```

Comment:
Thanks for this! Why we add do ee_transform() instead of doing ee_transform?

Reply:
I had to modify ee_transform to be a function instead of a property, so that we can index different end effectors (for cases with robots/agents with multiple arms). See [here](https://github.com/facebookresearch/habitat-lab/blob/557154aa990d4dca030e50e160f5b63a6dde580a/habitat-lab/habitat/articulated_agents/manipulator.py#L248).

Reply:
Should we rename it to get_ee_transform so we know it is doing a computation ?

Reply:
As I said above, we should keep this a property to maintain backward compatibility. I think ideal scenario would be to have it as a property and list which returns index 0 by default.

Reply:
Should we have get_ee_transform(ee_index) and the property ee_transform call get_ee_transform(0) ?